test/api_container_wait_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/test/request"
+
+	"github.com/go-check/check"
+)
+
+// APIContainerWaitSuite is the test suite for container wait API.
+type APIContainerWaitSuite struct{}
+
+func init() {
+	check.Suite(&APIContainerWaitSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *APIContainerWaitSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	PullImage(c, busyboxImage)
+}
+
+// TestWaitOk tests waiting a stopped container return 200.
+func (suite *APIContainerWaitSuite) TestWaitOk(c *check.C) {
+	cname := "TestWaitOk"
+
+	CreateBusyboxContainerOk(c, cname)
+	StartContainerOk(c, cname)
+	StopContainerOk(c, cname)
```

Comment:
Can we add some cases about running container? I think the common case is using for the `running` container

Reply:
Solved